package.json

```diff
@@ -309,15 +309,6 @@
 				"command": "cosmosDB.copyConnectionString",
 				"title": "Copy Connection String"
 			},
-			{
-				"category": "Cosmos DB",
```

Comment:
I don't think this will work - we need to define commands here if we're going to reference them from a menu time. Instead, you want to add this command to the "commandPalette" section with `"when": "never"`

Take a look at this: https://github.com/microsoft/vscode-cosmosdb/blob/master/package.json#L689

Reply:
VS Code should look for command in `menu` with command id which is registered by `vscode.commands.registerCommand`, and there's no need to define that in `commandPalette`. However, I haven't tested that on my side, have you tried to just remove the command from `commandPalette`? Did you meet any problem with that?

Reply:
Did you trying F5-ing the extension with your change? This is what I see:
 ![Screen Shot 2019-05-24 at 10 14 44 AM](https://user-images.githubusercontent.com/11282622/58345266-c87f3d00-7e0c-11e9-9670-b39d112766af.png)

Reply:
I just tried to debug, and got the same error. I don't think that make sense :-S

I have pushed another commit to set never show `cosmosDB.update` in command palette.